src/rest_api.py

```diff
@@ -452,7 +453,13 @@ def get_stacks_report(report):
     A report matching with the filename retrieved using the /stacks-report/list/{frequency}
     will be returned.
     """
-    return flask.jsonify(_s3_helper.get_object_content(report))
+    try:
+        return flask.jsonify(_s3_helper.get_object_content(report)), 200
+    except ClientError as e:
+        return flask.jsonify({
```

Comment:
Custom error should also be fine. I am ok with this approach, since it is internal to platform.